src/test/cache/Cache.basicCache.cpp

```diff
@@ -0,0 +1,253 @@
+#include <gtest/gtest.h>
```

Comment:
These filenames are a bit weird, I'd use underscores instead of dots.

Reply:
this is out of scope for this PR. I'll separate tests out into a testing PR.